Source/WebCore/css/CSSFontFace.cpp

```diff
@@ -153,8 +153,8 @@ void CSSFontFace::setFamilies(CSSValueList& family)
     RefPtr oldFamily = std::exchange(m_families, &family);
     mutableProperties().setProperty(CSSPropertyFontFamily, family);
 
-    iterateClients(m_clients, [&](CSSFontFaceClient& client) {
-        client.fontPropertyChanged(*this, oldFamily.get());
+    iterateClients(m_clients, [&, protectedThis = Ref { *this }](CSSFontFaceClient& client) {
```

Comment:
In this case a slightly nicer fix is NOESCAPE:

`template<typename T> void iterateClients(WeakHashSet<CSSFontFaceClient>& clients, **NOESCAPE** T callback)`

Reply:
There's no real need to update all callers to refcount their captures because we only use them locally, so standard enforcement of local variable safety also ensures capture safety.

Reply:
Thanks!

Reply:
@geoffreygaren it seems NOESCAPE can only be used for pointers. So the callback would have to be or decay to a function pointer, but then we couldn't capture. Should I go back to protected this?

` error: 'noescape' attribute only applies to pointer arguments [-Werror,-Wignored-attributes]`

Reply:
I provided a solution for this one.